beesdoo_website_eater/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+
+# Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'BEES coop Website Eater',
+
+    'summary': """
+    Show the eaters of a cooperator in the website portal.
+    """,
+    'description': """
+    """,
+
+    'author': 'Rémy Taymans',
+    'license': 'AGPL-3',
+    'version': '12.0.1.0',
```

Comment:
[Bump module version to 12.0.1.0.0](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration)